packages/app/features/dashboard/profile/edit/index.tsx

```diff
@@ -117,6 +131,20 @@ export default function EditProfileScreen() {
               onChangeText={handleChange("username")}
               icon={PersonOutline}
             />
+            <Line />
+            <FormInputWithIcon
+              value={values.twitter ?? ""}
+              placeholder="Twitter username"
+              onChangeText={handleChange("twitter")}
+              icon={Twitter}
+            />
+            <Line />
+            <FormInputWithIcon
```

Comment:
`FormInputWithIcon` has an error prop. Can we use it and remove displaying errors at the bottom?